src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerGovernmentOrPublicAuthorityDto.java

```diff
@@ -137,6 +154,38 @@ public void setNonLegalFirmMembersNatureOfControlTypes(List<NatureOfControlType>
         this.nonLegalFirmMembersNatureOfControlTypes = nonLegalFirmMembersNatureOfControlTypes;
     }
 
+    public List<NatureOfControlType> getNonLegalFirmControlNatureOfControlTypes() {
+        return nonLegalFirmControlNatureOfControlTypes;
+    }
+
+    public void setNonLegalFirmControlNatureOfControlTypes(List<NatureOfControlType> nonLegalFirmControlNatureOfControlTypes) {
+        this.nonLegalFirmControlNatureOfControlTypes = nonLegalFirmControlNatureOfControlTypes;
+    }
+
+    public List<NatureOfControlType> getTrustNatureOfControlTypes() {
```

Comment:
getter and setter need TrustControl added

Reply:
See question above.

Reply:
updated